CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+We gratefully accept bug reports and contributions from the community. There are some requirements we need to fulfill in order to be able to integrate contributions:
+
+ - As with any open source project, contributions will be reviewed by the project team and community and may need some modifications to be accepted.
+ - The contribution should not break API or ABI, unless there is a real justification for that. If there is an API change, the contribution, if accepted, will be merged only when there will be a major release.
+
+### Contributor License Agreement ( CLA )
```

Comment:
Please remove spaces after `(` and before `)`

Reply:
Normally, the number of `#` characters at the front means the hierarchy of the document. So for example, this would be element 1.1.1. However, there are no top-level heading, so I would suggest to use only 1 `#` if there is no heading about this. Also, note that the main mbed TLS readme is actually using `-` and `=` characters for the top-level headings:
https://raw.githubusercontent.com/ARMmbed/mbedtls/development/README.md